int16 R_Front #kuadran 1
int16 L_Front #kuadran 2
int16 L_Back  #kuadran 3
int16 R_Back  #kuadran 4
